Assets/Scripts/Augment/GunController.cs

```diff
@@ -36,8 +37,23 @@ private void FixedUpdate()
         }
     }
 
+    /// <summary>
+    /// Expects a percentage of ammunition to be reloaded.
+    /// This percentage is normalized eg. min = 0, max = 1.
```

Comment:
Ummmm so it's just a fraction. No need to call it a percentage when it isn't one.

Reply:
Ye, good point